### PR TITLE
DAG to setup nci database in sandbox environment

### PR DESCRIPTION
**Pull request by NikitaGandhi**
DAG to setup nci database in sandbox environment.

### CODE REVIEW

File: dags/k8s_nci_db_sync.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    schedule_interval=timedelta(days=7),
+)
+
+affinity = {
+    "nodeAffinity": {
+        "requiredDuringSchedulingIgnoredDuringExecution": {
+            "nodeSelectorTerms": [{
+                "matchExpressions": [{
+                    "key": "nodetype",
+                    "operator": "In",
```
Comment (whatnick) [inline dags/k8s_nci_db_sync.py:80]: We specifically do not want a spot node.

File: dags/k8s_nci_db_sync.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        "S3_KEY": S3_KEY,
+        "DB_HOSTNAME": DB_HOSTNAME,
+        "DB_DATABASE": DB_DATABASE,
+        "DB_PORT": "5432",
+        "BACKUP_PATH": BACKUP_PATH
+    },
+    # Use K8S secrets to send DB Creds
+    # Lift secrets into environment variables for datacube
+    "secrets": [
+        Secret("env", "DB_USERNAME", "explorer-admin", "postgres-username"),
```
Comment (pindge) [inline dags/k8s_nci_db_sync.py:54]: is there a copy of `explorer-admin` secret in `processing`?

Comment (NikitaGandhi) [reply dags/k8s_nci_db_sync.py:54]: yes. we do - https://bitbucket.org/geoscienceaustralia/datakube/src/00b14b3efa24d4447a2f2371667b66a9a5e7198a/workspaces/dea-sandbox/03_odc_k8s_apps/db_creds.tf#lines-120

**Review by pindge — APPROVED** [review]

**Comment by pindge** [comment]
Ready to go

**Pull request merged**